src/rewriters/DivModRewriter.h

```diff
@@ -0,0 +1,111 @@
+//
+// Created by Martin Blicha on 03.02.21.
+//
+
+#ifndef OPENSMT_DIVMODREWRITER_H
+#define OPENSMT_DIVMODREWRITER_H
+
+#include "PTRef.h"
+
+#include "Rewriter.h"
+
+#include "OsmtApiException.h"
+
+#include <unordered_map>
+#include <string>
+
+class LIALogic;
+
+class DivModConfig : public DefaultRewriterConfig {
+    LIALogic & logic;
+
+    struct DivModPair {
+        PTRef div;
+        PTRef mod;
+    };
+
+    std::unordered_map<std::pair<PTRef, PTRef>, DivModPair, PTRefPairHash> divModCache;
+    vec<PTRef> definitions;
+
+    const char * divPrefix = ".div";
+    const char * modPrefix = ".mod";
```

Comment:
Define the strings in LIATheory?

Reply:
I think this would create a dependency of `DivModRewriter` on `LIATheory` that does not really make sense. I would keep the rewriter as much independent as possible.
If these were to move somewhere, I would move them to `LIALogic`, that dependency already exists and is necessary.

Reply:
Ok, maybe we could move them there.

Reply:
I am not sure anymore about moving them out of here.
Do we plan to use them anywhere else?
I now see it more like an internal thing of the rewriter.

Reply:
Ok, as discussed, let's go with the current solution.